platform/security/certificate/security-ocsp-checker/src/main/java/org/codice/ddf/security/ocsp/checker/OcspChecker.java

```diff
@@ -123,127 +144,254 @@ Certificate convertToBouncyCastleCert(X509Certificate cert) throws OcspCheckerEx
+        AccessController.doPrivileged(
+            (PrivilegedAction<String>) SecurityConstants::getTruststorePassword);
+
+    if (isBlank(trustStorePath) || isBlank(trustStorePass)) {
+      throw new OcspCheckerException("Problem retrieving truststore properties.");
+    }
+
+    KeyStore truststore;
+
+    try (InputStream truststoreInputStream = new FileInputStream(trustStorePath)) {
+      truststore = SecurityConstants.newTruststore();
+      truststore.load(truststoreInputStream, trustStorePass.toCharArray());
+    } catch (CertificateException | IOException | KeyStoreException | NoSuchAlgorithmException e) {
+      throw new OcspCheckerException(
+          String.format("Problem loading truststore on path %s", trustStorePath));
+    }
+
     try {
-      SecureCxfClientFactory cxfClientFactory =
-          factory.getSecureCxfClientFactory(ocspServerUrl, WebClient.class);
-      WebClient client =
-          cxfClientFactory
-              .getWebClient()
-              .accept("application/ocsp-response")
-              .type("application/ocsp-request");
+      return getCertFromTruststoreWithX500Name(issuerName, truststore);
+    } catch (OcspCheckerException e) {
+      throw new OcspCheckerException("Problem finding issuer in truststore.", e);
+    }
+  }
 
-      return client.post(ocspReq.getEncoded());
+  /**
+   * Returns an {@link X509CertificateHolder} containing the issuer of the given {@param name}.
+   * Search is performed in the given {@param truststore}.
+   *
+   * @param name - the {@link X500Name} of the issuer.
+   * @param truststore - the {@link KeyStore} to check.
+   * @return {@link X509CertificateHolder} of the certificate with the given {@param name}.
+   * @throws OcspCheckerException if the {@param name} cannot be found in the {@param truststore}.
+   */
+  private X509CertificateHolder getCertFromTruststoreWithX500Name(
+      X500Name name, KeyStore truststore) throws OcspCheckerException {
+    Enumeration<String> aliases;
 
-    } catch (ProcessingException | IOException e) {
-      postErrorEvent(
-          "Unable to send OCSP request. The certificate status could not be verified. "
-              + e.getMessage());
-      throw new OcspCheckerException();
+    try {
+      aliases = truststore.aliases();
+    } catch (KeyStoreException e) {
+      throw new OcspCheckerException("Problem getting aliases from truststore.", e);
+    }
+
+    while (aliases.hasMoreElements()) {
+      String currentAlias = aliases.nextElement();
+
+      try {
+        java.security.cert.Certificate currentCert = truststore.getCertificate(currentAlias);
+        X509CertificateHolder currentCertHolder =
+            new X509CertificateHolder(currentCert.getEncoded());
+        X500Name currentName = currentCertHolder.getSubject();
+        if (name.equals(currentName)) {
+          return currentCertHolder;
+        }
+      } catch (CertificateEncodingException | IOException | KeyStoreException ignore) {
+      }
```

Comment:
❗️ If there's an exception being thrown, we should be at least logging information about the exception. Otherwise, there's no way to troubleshoot what's happening without attaching a debugger.